styles/style.css

```diff
@@ -1,15 +1,454 @@
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, 
-figure, figcaption, footer, header, nav, section, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+
+.c-red {
+  color: #f72210;
+}
+
+.bg-red {
+  background-color: #f72210;
+  color: white;
+}
+
+main.container-fluid {
+  max-width: 1330px;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+/* nav */
+
+#nav-lg {
+  height: 108px;
+  padding: 20px 10px 15px;
+}
+
+.bottom-left div {
+  background-color: rgb(241, 241, 241);
+}
+
+.v-grid div:not(:last-child) {
+  border-bottom: 1px solid rgb(211, 210, 210);
+}
+
+.newsletter > div > div:last-child {
+  border: 1px solid rgb(209, 209, 209);
+}
+
+#nav-lg > div {
+  padding: 0 20px;
+}
+
+#main-logo {
+  width: 400px;
+  fill: white;
+  height: 60px;
+}
+
+#nav-lg-left div {
+  line-height: 2;
+  font-size: 15px;
+}
+
+footer img {
+  width: 240px;
+}
+
+.arrow > img {
+  padding-left: 10px;
+  height: 13px;
+}
+
+.nav-md-left img {
+  width: 170px;
+}
+
+.nav-sm-left img {
+  width: 165px;
+}
+
+.showcase-mini-story img {
+  float: left;
+  margin-right: 15px;
+}
+
+.editor-picks-article img {
+  width: 100%;
+}
+
+.show-case-article a img {
+  width: 100%;
+}
+
+.v-grid div img {
+  float: left;
+}
+
+.different-articles-container > div > img {
+  width: 100%;
+}
+
+.bottom-left-img img:first-child {
+  z-index: 2;
+  left: 80px;
+  top: 15px;
+}
+
+.different-articles-container img:hover {
+  cursor: pointer;
+}
+
+.showcase-mini-story img:hover {
+  cursor: pointer;
+}
+
+.v-grid div:first-child img {
+  width: 96px;
+  height: 65px;
+}
+
+a {
+  color: black;
+  font-weight: bold;
+}
+
+h4 a {
+  color: black;
+  padding-top: 10px;
+}
+
+.different-articles-title a {
+  font-size: 19px;
+}
+
+#nav-lg-left img {
+  height: 20px;
+}
+
+h4 a:hover {
+  text-decoration: underline red;
+  color: black;
+}
+
+.footer-bottom ul li a {
+  color: rgb(107, 107, 107);
+  padding: 0 10px;
+  border-right: rgb(206, 206, 206) 1px solid;
+}
+
+.different-articles-title a:hover {
+  color: #f72210;
+  text-decoration: none;
+}
+
+.v-grid div a:hover {
+  text-decoration: underline red;
+  color: black;
+}
+
+#nav-md,
+#nav-sm {
+  background-color: #f72210;
+  padding: 5px 0;
+}
+
+#nav-sm {
+  padding: 3px 0;
+}
+
+.nav-md-right a > i,
+.nav-sm-right a > i {
+  color: rgba(233, 233, 233, 0.5);
+}
+
+.nav-md-right > div > *,
+.nav-sm-right > div > * {
+  margin-left: 15px;
+}
+
+.nav-sm-right button {
+  font-size: 12px;
+}
+
+div#nav-list {
+  margin: -10px 0 0;
+  z-index: 2;
+  padding-top: 15px;
+}
+
+#nav-list ul {
+  width: 87%;
+}
+
+#nav-list li {
+  flex-grow: 1;
+}
+
+.footer-top-left p {
+  font-size: 14px;
+  color: rgb(134, 134, 134);
+}
+
+.showcase-mini-grid figcaption p {
+  font-weight: 500;
+  color: gray;
+}
+
+.footer-bottom ul:first-child li:first-child a,
+.footer-bottom ul:first-child li:last-child a,
+.footer-bottom ul:last-child li:last-child a {
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<a>` element` and delete the left unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_